From afc5ba9e94f86a11d50f3554efeafd402faddacb Mon Sep 17 00:00:00 2001
From: "Laurence J. Lane" <ljlane@debian.org>
Date: Mon, 2 Sep 2013 16:46:50 -0400
Subject: [PATCH] iptables: mention iptables-reply in SEE ALSO

Add iptables-apply(8) to the SEE ALSO section of *-save(8)
and *-restore(8).

References: http://bugs.debian.org/660748

Signed-off-by: Laurence J. Lane <ljlane@debian.org>
---
 iptables/iptables-restore.8.in | 2 +-
 iptables/iptables-save.8.in    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: pkg-iptables/iptables/iptables-restore.8.in
===================================================================
--- pkg-iptables.orig/iptables/iptables-restore.8.in
+++ pkg-iptables/iptables/iptables-restore.8.in
@@ -87,7 +87,7 @@ from Rusty Russell.
 .br
 Andras Kis-Szabo <kisza@sch.bme.hu> contributed ip6tables-restore.
 .SH SEE ALSO
-\fBiptables\-save\fP(8), \fBiptables\fP(8)
+\fBiptables\-apply\fP(8),\fBiptables\-save\fP(8), \fBiptables\fP(8)
 .PP
 The iptables-HOWTO, which details more iptables usage, the NAT-HOWTO,
 which details NAT, and the netfilter-hacking-HOWTO which details the
Index: pkg-iptables/iptables/iptables-save.8.in
===================================================================
--- pkg-iptables.orig/iptables/iptables-save.8.in
+++ pkg-iptables/iptables/iptables-save.8.in
@@ -59,7 +59,7 @@ Rusty Russell <rusty@rustcorp.com.au>
 .br
 Andras Kis-Szabo <kisza@sch.bme.hu> contributed ip6tables-save.
 .SH SEE ALSO
-\fBiptables\-restore\fP(8), \fBiptables\fP(8)
+\fBiptables\-apply\fP(8),\fBiptables\-restore\fP(8), \fBiptables\fP(8)
 .PP
 The iptables-HOWTO, which details more iptables usage, the NAT-HOWTO,
 which details NAT, and the netfilter-hacking-HOWTO which details the
